src/java.base/share/classes/java/lang/foreign/ValueLayout.java

```diff
@@ -740,5 +738,102 @@ public boolean isUnbounded() {
      * MemoryLayout.valueLayout(double.class, ByteOrder.nativeOrder()).withBitAlignment(64);
      * }
      */
-    public static final OfDouble JAVA_DOUBLE = new OfDouble(ByteOrder.nativeOrder()).withBitAlignment(64);
+    public static final OfDouble JAVA_DOUBLE = new OfDouble(ByteOrder.nativeOrder());
+
+    /**
+     * An unaligned value layout constant whose size is the same as that of a machine address ({@code size_t}),
+     * and byte order set to {@link ByteOrder#nativeOrder()}.
+     * Equivalent to the following code:
+     * {@snippet lang=java :
+     * MemoryLayout.valueLayout(MemorySegment.class, ByteOrder.nativeOrder())
+     *             .withBitAlignment(8);
```

Comment:
An alternative snippet is to leverage the exiting aligned constant and make it unaligned, like what we would commonly do in internal code:
```
ADDRESS.withBitAlignment(8);
```
I think expresses the intent more clearly.